test/jdk/java/sql/testng/test/sql/TimestampTests.java

```diff
@@ -642,6 +642,35 @@ public void test52(long value, String ts) {
         assertEquals(ts1.toString(), ts, "ts1.toString() != ts");
     }
 
+    @Test
+    public void test53() {
+        // The latest Instant that can be converted to a Timestamp.
+        Instant instant1 = Instant.ofEpochSecond(Long.MAX_VALUE / 1000, 999_999_999);
+        assertEquals(instant1, Timestamp.from(instant1).toInstant());
```

Comment:
The 1st arg to `assertEquals()` should be the actual value, the 2nd should be the expected result. I guess you expect `instant1` to be the expected result.
(TestNG and JUnit have opposite conventions...)

Reply:
Thanks, I hadn't realized that.